deeplearning4j-nn/src/main/java/org/deeplearning4j/nn/layers/BaseLayer.java

```diff
@@ -76,8 +76,8 @@ public BaseLayer(NeuralNetConfiguration conf) {
     }
 
     public BaseLayer(NeuralNetConfiguration conf, INDArray input) {
-        this(conf);
         this.input = input;
+        this.conf = conf;
```

Comment:
Thanks! Missed it somehow during conflicts resolution... I've backported preOutput cache to master, and then merge conflict appeared.